### ISSUE TITLE
FreeType throws exception when parameter "incremental" is true.

### ISSUE THREAD

**Issue by 9Livees** [libgdx/libgdx#3270]
After updating libgdx from version 1.6.1 to 1.6.3 such code:
```
import com.badlogic.gdx.ApplicationAdapter;
import com.badlogic.gdx.Gdx;
import com.badlogic.gdx.backends.lwjgl.LwjglApplication;
import com.badlogic.gdx.graphics.g2d.BitmapFont;
import com.badlogic.gdx.graphics.g2d.freetype.FreeTypeFontGenerator;

public class Example extends ApplicationAdapter {
    public void create () {
        FreeTypeFontGenerator generator = new FreeTypeFontGenerator(Gdx.files.internal("font.ttf"));
        FreeTypeFontGenerator.FreeTypeFontParameter parameter = new FreeTypeFontGenerator.FreeTypeFontParameter();
        parameter.incremental = true;
        BitmapFont font = generator.generateFont(parameter); 
        generator.dispose();
    }

    public static void main (String[] args) throws Exception {
        new LwjglApplication(new Example());
    }
}
```
throws exception:
```
Exception in thread "LWJGL Application" java.lang.NullPointerException
	at com.badlogic.gdx.graphics.g2d.freetype.FreeTypeFontGenerator$FreeTypeBitmapFontData.getGlyph(FreeTypeFontGenerator.java:610)
	at com.badlogic.gdx.graphics.g2d.freetype.FreeTypeFontGenerator.generateData(FreeTypeFontGenerator.java:410)
	at com.badlogic.gdx.graphics.g2d.freetype.FreeTypeFontGenerator.generateFont(FreeTypeFontGenerator.java:163)
	at com.badlogic.gdx.graphics.g2d.freetype.FreeTypeFontGenerator.generateFont(FreeTypeFontGenerator.java:156)
	at Example.create(Example.java:12)
	at com.badlogic.gdx.backends.lwjgl.LwjglApplication.mainLoop(LwjglApplication.java:143)
	at com.badlogic.gdx.backends.lwjgl.LwjglApplication$1.run(LwjglApplication.java:120)
```

I tried to use different TTF fonts, but in all cases it throws NullPointerException.

**Comment by NathanSweet** [comment]
See FreeTypeFontParameter `incremental` javadoc.

**Comment by 9Livees** [comment]
Thanks for help. But in this case exception is thrown before calling ```generator.dispose()``` (If you mean that FreeTypeFontGenerator must not be disposed until the font is no longer needed). And this problem occurs only from version 1.6.3.

**Comment by NathanSweet** [comment]
Sorry, I should have tried your SSCCE!

**Comment by NathanSweet** [comment]
Fixed in 152b5a66e0cf7bbf970b3b756c28ce915d977b22
